### PR TITLE
Remove using createCore from documentation, tests and examples. Fix code style in examples

### LINKED ISSUE #41: Fix tests, doc and electron example after removing createCore API
After #39 merged, the `createCore` API is removed. There are some issues:

- `npm test` fails, the test cases need to be fixed.

- The API [doc](https://github.com/intel/inference-engine-node/blob/master/doc/api.md) also to be fixed.

- The [electron example](https://github.com/intel/inference-engine-node/tree/master/example/hello_classification_electron) needs to be fixed.

@artyomtugaryov , please help. CC @lionkunonly .

### PR DESCRIPTION
**Pull request by artyomtugaryov**

### CODE REVIEW

File: example/hello_classification_electron/static/lib/stats.customized.min.js [hunk 1]
```diff
@@ -1 +1,41 @@
-(function(global,factory){typeof exports==='object'&&typeof module!=='undefined'?module.exports=factory():typeof define==='function'&&define.amd?define(factory):(global.Stats=factory())}(this,(function(){'use strict';var fps=document.querySelector('#fps');var Stats=function(){var beginTime=(performance||Date).now(),prevTime=beginTime,frames=0;var fpsPanel=new Stats.Panel();return{begin:function(){beginTime=(performance||Date).now()},end:function(){frames++;var time=(performance||Date).now();if(time>prevTime+1000){fpsPanel.update((frames*1000)/(time-prevTime),100);prevTime=time;frames=0}return time},update:function(){beginTime=this.end()}}};Stats.Panel=function(){var min=Infinity,max=0,round=Math.round;return{update:function(value,maxValue){min=Math.min(min,value);max=Math.max(max,value);var fpsnow=round(value);fps.innerHTML=fpsnow+' FPS'}}};return Stats})));
\ No newline at end of file
+(function(global, factory) {
```
Comment (huningxin) [inline example/hello_classification_electron/static/lib/stats.customized.min.js]: I suppose this file was imported, so we can ignore the style of it.

**Review by huningxin — APPROVED** [review]
It looks good with a nit. Thanks @artyomtugaryov !

File: example/hello_classification_electron/renderer.js [hunk 2]
```diff
@@ -3,7 +3,6 @@ const fs = require('fs').promises;
 const canvasElement = document.getElementById('canvas');
 const videoElement = document.getElementById('video');
 const imageElement = document.getElementById('image');
-const progressBar = document.getElementById('progressBar');
```
Comment (lionkunonly) [inline example/hello_classification_electron/renderer.js]: The `progressBar` is a UI  element for the Electron Example, please keep it.

Comment (artyomtugaryov) [reply example/hello_classification_electron/renderer.js]: Ok

**Review by lionkunonly — APPROVED** [review]

File: test/inference_engine.js [hunk 3]
```diff
@@ -45,15 +43,11 @@ describe('InferenceEngine Test', () => {
     expect(apiVersion.minor).to.be.a('number');
   });
 
-  it('InferenceEngine.createCore should be a function', () => {
-    expect(ie.createCore).to.be.a('function');
+  it('new InferenceEngine.Core should return a Core object', () => {
```
Comment (huningxin) [inline test/inference_engine.js]: 'InferenceEngine.Core should be a function'

File: test/inference_engine.js [hunk 4]
```diff
@@ -45,15 +43,11 @@ describe('InferenceEngine Test', () => {
...
+    expect(new ie.Core()).to.be.a('Core');
```
Comment (huningxin) [inline test/inference_engine.js]: expect(ie.Core).to.be.a('function')

File: test/inference_engine.js [hunk 5]
```diff
@@ -45,15 +43,11 @@ describe('InferenceEngine Test', () => {
...
   });
 
-  it('InferenceEngine.createCore should return a Core object', () => {
-    expect(ie.createCore()).to.be.a('Core');
+  it('InferenceEngine.Core should return a Core object', () => {
```
Comment (huningxin) [inline test/inference_engine.js]: 'new InferenceEngine.Core() should return a Core object'

**Review by huningxin — COMMENTED** [review]
Some comments for tests, please take a look. Thanks.

**Comment by huningxin** [comment]
Looks good. Thanks @artyomtugaryov !

**Pull request merged**